sycl/test-e2e/KernelAndProgram/disable-caching.cpp

```diff
@@ -6,6 +6,7 @@
 // RUN: | FileCheck %s
 // RUN: env ZE_DEBUG=-6 SYCL_PI_TRACE=-1 %{run} %t.out \
 // RUN: | FileCheck %s --check-prefixes=CHECK-CACHE
+// REQUIRES: aspect-usm_shared_allocations
```

Comment:
Can we rewite this test case to use buffer/accessors instead? I don't see how that change will alter the caching/eviction of kernels/programs.

Reply:
I've changed it to use malloc_device because with buffer/accessor the test was failing due to some PI calls not showing up as expected.